forms-flow-web/src/components/Form/Item/Submission/Item/Edit.js

```diff
@@ -49,6 +50,16 @@ const Edit = React.memo((props) => {
     }
   }, [applicationStatus, userRoles, dispatch, submissionId, formId, onFormSubmit ]);
 
+  let scrollToErrorInterval = null;
+  useEffect(() => {
+      scrollToErrorInterval = setInterval(() => {
```

Comment:
I think we might be able to use either `useEffect` (or potentially `useCallback` since we're dealing with a ref) here to trigger the listener setup instead of an interval

Reply:
Well, I tried different combinations of `useEffect` and `useCallback`. The problem is that even if after listening to `refForm`, it becomes available (i.e. not undefined), still the formio instance inside `refForm.current` if undefined at least after 1s. If my code touches that before, the form won't be responsive anymore and the user has to refresh the page.

Apart from that, I see some weird behavior in the `useEffect` in this project, not sure what's different from other React apps that adding a dependency to `useEffect` or `useCallback` won't work as expected, other words, the call back doesn't trigger when a dependency changes, like in other React apps.